<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CustomerWindow</class>
 <widget class="QMainWindow" name="CustomerWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>413</width>
    <height>292</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget {
    background-color: black;
    color: white;
}
/* Customize QPushButton specifically */
QPushButton {
    background-color: gray; /* A slightly lighter background for buttons */
    color: white;
    border: 2px solid white;
    border-radius: 5px;
}

/* Customize QTabWidget */
QTabWidget::pane {
    background-color: black;
    border: 1px solid white;
}

QTabWidget::tab-bar {
    alignment: center;
}

QTabBar::tab {
    background: gray;
    color: white;
    padding: 5px;
    border: 1px solid white;
}

QTabBar::tab:selected {
    background: white;
    color: black;
}

QLineEdit {
    border: 1px solid gray;
    border-radius: 3px;
    padding: 2px;
}
font: 900 italic 9pt &quot;Segoe UI&quot;;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>391</width>
      <height>251</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>AddP</string>
     </attribute>
     <widget class="QWidget" name="verticalLayoutWidget_3">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>0</y>
        <width>191</width>
        <height>211</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Name:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="aName"/>
       </item>
       <item>
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>Brand:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="aBrand"/>
       </item>
       <item>
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>Type:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="aType"/>
       </item>
       <item>
        <widget class="QLabel" name="label_8">
         <property name="text">
          <string>Size:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="aSize"/>
       </item>
       <item>
        <widget class="QLabel" name="label_9">
         <property name="text">
          <string>Price:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="aPrice"/>
       </item>
      </layout>
     </widget>
     <widget class="QPushButton" name="aButton">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>90</y>
        <width>80</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string>Add</string>
      </property>
     </widget>
     <widget class="QLabel" name="aResult">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>120</y>
        <width>71</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Result...</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>EditP</string>
     </attribute>
     <widget class="QWidget" name="verticalLayoutWidget">
      <property name="geometry">
       <rect>
        <x>109</x>
        <y>10</y>
        <width>161</width>
        <height>191</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Enter Product ID:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="eID"/>
       </item>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Product Component To Change:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="eComponents">
         <item>
          <property name="text">
           <string>Size</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Price</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Brand</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Type</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Name</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>New Component Value:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="eNew"/>
       </item>
       <item alignment="Qt::AlignmentFlag::AlignHCenter">
        <widget class="QPushButton" name="eEnter">
         <property name="minimumSize">
          <size>
           <width>50</width>
           <height>0</height>
          </size>
         </property>
         <property name="text">
          <string>Enter</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="eResult">
         <property name="text">
          <string>Result...</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>RemoveP</string>
     </attribute>
     <widget class="QWidget" name="verticalLayoutWidget_2">
      <property name="geometry">
       <rect>
        <x>99</x>
        <y>10</y>
        <width>181</width>
        <height>91</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Enter ID For Product To Be Removed:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="rID"/>
       </item>
       <item alignment="Qt::AlignmentFlag::AlignHCenter">
        <widget class="QPushButton" name="rButton">
         <property name="minimumSize">
          <size>
           <width>60</width>
           <height>0</height>
          </size>
         </property>
         <property name="text">
          <string>Enter</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="rResult">
         <property name="text">
          <string>Result...</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>413</width>
     <height>17</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuEcomLIq">
    <property name="title">
     <string>EcomLIq</string>
    </property>
   </widget>
   <addaction name="menuEcomLIq"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
